utils/setUser.js

```diff
@@ -0,0 +1,18 @@
+import jwtDecode from 'jwt-decode';
+import store from '@Redux/store';
+import { authSuccess } from '@Actions/authActions';
+
+const setUser = () => {
+  if (localStorage.jwtToken) {
+    const decoded = jwtDecode(localStorage.jwtToken);
+    store.dispatch(authSuccess(decoded));
+
+    const currentTime = Date.now() / 1000;
+    if (decoded.exp < currentTime) {
+      // dispatch logout function here;
```

Comment:
But did you dispatch the logout function here?

Reply:
I did this before updating my branch, I'll dispatch the logout function here.